Dockerfile

```diff
@@ -8,31 +8,39 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 RUN apt-get -qq update \
   && apt-get install -y --no-install-recommends \
-    apt-transport-https \
-    build-essential \
-    curl \
-    dirmngr \
-    gnupg \
-    git \
-    ruby-full \
-    twine \
+  apt-transport-https \
+  build-essential \
+  curl \
+  dirmngr \
+  gnupg \
+  git \
+  ruby-full \
+  twine \
+  jq \
+  unzip \
   && curl -fsSL https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -o /tmp/packages-microsoft-prod.deb \
   && dpkg -i /tmp/packages-microsoft-prod.deb \
   && rm /tmp/packages-microsoft-prod.deb \
   && echo 'deb [arch=amd64] https://download.docker.com/linux/debian buster stable' >> /etc/apt/sources.list \
   && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
   && apt-get update -qq \
   && apt-get install -y --no-install-recommends \
-    dotnet-sdk-3.1 \
-    docker-ce-cli \
+  dotnet-sdk-3.1 \
+  docker-ce-cli \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/* \
   && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s --  --profile minimal -y \
   && cargo --version \
   && cargo install cargo-hack \
   # Stick with 3.1.x as 3.2.x doesn't install on Debian Buster for some reason
   && gem update --no-document --system 3.1.5 \
-  && gem install cocoapods
+  && gem install cocoapods \
+  # Install https://github.com/getsentry/symbol-collector
+  && symbol_collector_url=$(curl -s https://api.github.com/repos/getsentry/symbol-collector/releases/tags/1.3.1 | \
+  jq -r '.assets[].browser_download_url | select(endswith("symbolcollector-console-linux-x64.zip"))') \
+  && curl -sL $symbol_collector_url -o "/tmp/sym-collector.zip" \
+  && unzip /tmp/sym-collector.zip -d /usr/local/bin/ \
```

Comment:
I think you can just pipe this into `unzip` and avoid storing on disk?

Reply:
`unzip` has a bug:

> Archives read from standard input are not yet supported, except with `funzip` (and then only the first member of the archive can be extracted).

`funzip` has some limitations:

> and it extracts the first member from the archive to stdout

(note the _first member_ and _to stdout_.)

I haven't found any alternatives _without_ storing on disk and directly piping, so suggestions are welcomed.

Reply:
Oh, I wasn't aware of that. I guess I was thinking of `tar` as that's the more common format across Linux :)